client/scripts/views/pages/new_proposal/new_proposal_form.ts

```diff
@@ -329,9 +332,11 @@ const NewProposalForm = {
           hasAction && m(EdgewareFunctionPicker),
           hasTags
             && m(AutoCompleteTagForm, {
```

Comment:
Looks reused again here too

Reply:
Okay it's slightly weirder for the component to be reused across multiple files. I would be okay leaving it completely as is tbh